e2e/rook-test-framework/enums/rook_platform_types.go

```diff
@@ -0,0 +1,41 @@
+package enums
+
+import (
+	"errors"
+	"strings"
+)
+
+type RookPlatformType int
+
+const (
+	Kubernetes RookPlatformType = iota + 1
+	BareMetal
+	StandAlone
+	None
+)
+
+var platforms = [...]string{
+	"Kubernetes",
+	"BareMetal",
+	"StandAlone",
+	"None",
+}
+
+func (platform RookPlatformType) String() string {
+	return platforms[platform-1]
+}
+
+func GetRookPlatFormTypeFromString(name string) (RookPlatformType, error) {
+	switch {
+	case strings.EqualFold(name, Kubernetes.String()):
```

Comment:
`Kubernetes.String()` will return "1", correct? As I read this code I thought the intention would be to compare against the string "Kubernetes" instead of the string "1"